raphtory/src/algorithms/metrics/balance.rs

```diff
@@ -169,32 +173,32 @@ mod sum_weight_test {
 
         let res = balance(&graph, "value_dec".to_string(), Direction::BOTH, None);
         let expected = vec![
-            ("1".to_string(), -26.0),
-            ("2".to_string(), 7.0),
-            ("3".to_string(), 12.0),
-            ("4".to_string(), 5.0),
-            ("5".to_string(), 2.0),
+            ("1".to_string(), Some(-26.0)),
+            ("2".to_string(), Some(7.0)),
+            ("3".to_string(), Some(12.0)),
+            ("4".to_string(), Some(5.0)),
+            ("5".to_string(), Some(2.0)),
         ];
-        assert_eq!(res.sort_by_key(false), expected);
+        assert_eq!(res.sort_by_vertex_id(false), expected);
 
         let res = balance(&graph, "value_dec".to_string(), Direction::IN, None);
         let expected = vec![
-            ("1".to_string(), 6.0),
-            ("2".to_string(), 12.0),
-            ("3".to_string(), 15.0),
-            ("4".to_string(), 20.0),
-            ("5".to_string(), 2.0),
+            ("1".to_string(), Some(6.0)),
+            ("2".to_string(), Some(12.0)),
+            ("3".to_string(), Some(15.0)),
+            ("4".to_string(), Some(20.0)),
+            ("5".to_string(), Some(2.0)),
         ];
-        assert_eq!(res.sort_by_key(false), expected);
+        assert_eq!(res.sort_by_vertex_id(false), expected);
 
         let res = balance(&graph, "value_dec".to_string(), Direction::OUT, None);
         let expected = vec![
-            ("1".to_string(), -32.0),
-            ("2".to_string(), -5.0),
-            ("3".to_string(), -3.0),
-            ("4".to_string(), -15.0),
-            ("5".to_string(), 0.0),
+            ("1".to_string(), Some(-32.0)),
+            ("2".to_string(), Some(-5.0)),
+            ("3".to_string(), Some(-3.0)),
+            ("4".to_string(), Some(-15.0)),
+            ("5".to_string(), Some(0.0)),
         ];
-        assert_eq!(res.sort_by_key(false), expected);
+        assert_eq!(res.sort_by_vertex_id(false), expected);
```

Comment:
Lets go over these names on monday as well

Reply:
rename to sort_by_vertex